core/vm/contracts.go

```diff
@@ -50,10 +51,11 @@ type PrecompiledContract interface {
 // PrecompiledContractsHomestead contains the default set of pre-compiled Ethereum
 // contracts used in the Frontier and Homestead releases.
 var PrecompiledContractsHomestead = map[common.Address]PrecompiledContract{
```

Comment:
why homestead, not the later version?

Reply:
Originally this commit was for the iliad fork branch - we've since updated to be for the odyssey dev branch where as you rightly point out it should be on all forks. Sorry for the confusion